ansible/roles/mycar/files/custom/car/services/jobs/job_record.py

```diff
@@ -212,7 +212,7 @@ def start_training(self, remote_dataset_path):
                           training_job.job_id)
         self.logger.debug('[job_id: %i]  Telling the user it\'s model training is launched',
                           self.get_id())
-        self.display_screen_msg("Entrainement du modèle on attend un prof 🎓️... ")
+        self.display_screen_msg("UI-train | Entrainement du modèle on attend un prof 🎓️... ")
```

Comment:
We should keep this as it indicate that we don't have any AI Trainer available this is really useful